docs/CategoryGuides/MiscCategories/AllMajorEndings.md

```diff
@@ -379,7 +379,7 @@ After this, continue on to the Papyrus fight and kick his butt into the stratosp
 
 ### Waterfall
 
-Waterfall has a handfull of different routes that make effectively summarizing it difficult. [Here](/CategoryGuides/MainCategories/Genocide?id=Waterfall alt="USRC Guides - Genocide | Waterfall")  is the relevant section fro the geno guide.
+Waterfall has a handfull of different routes that make effectively summarizing it difficult. [Here](/CategoryGuides/MainCategories/Genocide?id=Waterfall)  is the relevant section fro the geno guide.
```

Comment:
I take it the Markdown format doesn't support alt-text, then. I see a fair amount of advice against linking the word "here"-- "relevant section from the Geno guide" would be more to the point, but also a much clumsier link due to its word count. Use your judgment; maybe "Waterfall is the relevant section from the Geno guide" with a link on Waterfall. Also, "from" is misspelled.